src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
```

Comment:
First of all, you should not create a `Lottery` object three times and name it "ball".
Instead, create one single lottery:
`Loterry loterry = new Lottery();`
See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes).

Then, use for loop for creating several objects of the same class. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class).

And don't use `toString()` method within the `System.out.println()`, it will be called automatically.

So, applying all the above, your code may change to this:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < RANDOM_BALLS_COUNT; i++) {
             System.out.println(lottery.getRandomBall());
        }
```

P.S. `RANDOM_BALLS_COUNT` is the constant, that has to be created on the class-level. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants).